src/backend/marsha/core/management/commands/transcript_video.py

```diff
@@ -0,0 +1,54 @@
+"""Management command to transcript a video."""
+
+from django.core.management import BaseCommand
+
+from marsha.core import defaults
+from marsha.core.models import TimedTextTrack, Video
+from marsha.core.utils.transcript_utils import transcript
+
+
+class Command(BaseCommand):
+    """Transcript a video."""
+
+    help = "Transcript a video"
+
+    def add_arguments(self, parser):
+        parser.add_argument("--video-id", type=str)
```

Comment:
```suggestion
        parser.add_argument("--video-id", type=uuid.UUID)
```

Reply:
When doing the command line to launch the django command, it think we will pass a string representing the UUID ?

Reply:
The `type` argument is not the constructor to use to cast the received argument. So yes the received argument will be a string then it will be cast as an uuid. So if the format is wrong, a `TypeError` will be raised.